app/services/meeting_starter.rb

```diff
@@ -21,7 +21,7 @@ def call
     begin
       meeting = BigBlueButtonApi.new.start_meeting room: @room, options: options, presentation_url: @presentation_url
 
-      @room.update!(last_session: DateTime.strptime(meeting[:createTime].to_s, '%Q'))
+      @room.update!(last_session: DateTime.strptime(meeting[:createTime].to_s, '%Q', online: true))
```

Comment:
This isn't in the right place, it should be after the `)`

Reply:
Embarassing